flask_tutorial-code_review/static/base_11.js

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
bad file name

Reply:
done, hope its good name

Reply:
try something more precise like sidebarFucs keep the conventions of camelCase for JavaScript

Reply:
done